# Molar Entropy of an Ideal Monatomic Gas
## Chem 113, Spring 2020, Liang Shi

In this notebook, we will calculate the molar entropies of noble gases using statistical mechanics.

The partition function of an ideal monatomic gas is given by

$$ Q = \frac{q^N}{N!}, $$

where the molecular partition function (electronic part is neglected) is given by

$$ q = \left ( \frac{2\pi m k_B T}{h^2} \right)^{3/2} V. $$

We also know entropy, $S$, can be computed from partition function, $Q$, via

$$ S = k_B T \left ( \frac{\partial \ln Q}{ \partial T} \right)_{NV} + k_B \ln Q. $$ 

Let's evaluate $\ln Q$ first:

$$ \ln Q = N \ln q - \ln N!$$

and then we have

$$ \left ( \frac{\partial \ln Q}{ \partial T} \right)_{NV} = N \frac{\partial \ln q}{ \partial T} = \frac{3N}{2T}.$$

Now entropy reads

$$ S = \frac{3}{2} k_B N + k_B N \ln \left [ \left ( \frac{2\pi m k_B T}{h^2} \right)^{3/2} V \right ] - k_B  \ln N!$$

Note that $N=n N_A$, $k_B = R/N_A$. With the Stirling approximation: for large $N$, $\ln N! \approx N \ln N - N$, we have

$$ S= \frac{3}{2} nR + nR \ln \left [ \left ( \frac{2\pi m k_B T}{h^2} \right)^{3/2} V \right ] - nR \ln N + nR $$

$$ S= \frac{5}{2} nR + nR \ln \left [ \left ( \frac{2\pi m k_B T}{h^2} \right)^{3/2} \frac{V}{N} \right ]. $$

Introducing molar entropy, $\bar{S}=S/n$, and molar volume, $\bar{V}=V/n$, the final expression is

$$ \bar{S} = \frac{S}{n} = \frac{5}{2} R + R \ln \left [ \left ( \frac{2\pi m k_B T}{h^2} \right)^{3/2} \frac{\bar{V}}{N_A} \right ] $$


** Let's do this calculation for Ar at 298.15 K and 1 bar.**

In [1]:
# load some libraries for numerical operations and plotting
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline

Compute the molar volume first using the ideal gas law ($P \bar{V} = RT $)

In [2]:
T = 298.15    # K
R = 0.0831451 # L bar /K /mol
P = 1.0       # bar
Vbar = R*T/P*0.001 # m^3 /mol (0.001 converts L to m^3)

Set other parameters (*we will use the SI units for the rest of the notebook.*)

In [3]:
# Parameters in SI units
R = 8.314510 # re-define R in the unit of J / K /mol
molar_mass = 39.948*0.001 # kg / mol (0.001 converts g to kg)
Na = 6.0221413e23 # Avogadro's number in /mol
kB = 1.380658e-23 # Boltzmann constant in J /K
h = 6.6260755e-34 # Planck constant in J s
m = molar_mass / Na # mass of an Ar atom

Compute molar entropy using the derived equation:

In [4]:
factor1 = 2.0*np.pi*m*kB*T/h/h
factor2 = Vbar / Na
s = 2.5*R + R*np.log(np.power(factor1,1.5)*factor2)
print(s)

154.8466488394353


**Group Exercise:**

Repeat the calculation above for other noble gases, and fill the form below.

| Gas  | molar mass | $\bar{S}$ (theory)  | $\bar{S}$ (exp.)  |
|--:|--:|--:|--:|
| He   | 4.003   |   | 126.153  |
| Ne   | 20.180  |   | 146.328  |
| Ar   | 39.948  | 154.847  | 154.846  |
| Kr   | 83.798  |   | 164.085  |
| Xe   | 131.293 |   | 169.685  |

Note: molar mass is in the unit of $g/mol$, and the molar entropy is in the unit of $J \cdot K^{-1} \cdot mol^{-1}$. The experimental data are taken from National Institute of Standards and Technology (NIST), U.S. Department of Commerce. 